nengo/tests/test_dists.py

```diff
@@ -79,6 +79,28 @@ def test_hypersphere_surface(dimensions, rng):
     assert np.allclose(np.mean(samples, axis=0), 0, atol=0.25 / dimensions)
 
 
+@pytest.mark.parametrize("low,high", [(-2, -1), (-1, 1), (1, 2), (1, -1)])
```

Comment:
What does it mean for low or high to be less than zero?  Doesn't that lead to imaginary numbers in the ```low ** (1/D)``` calculation?  Shouldn't there be a check that they are always great than zero?